ccd-definition/ComplexTypes/GeneralApplicationGAspec.json

```diff
@@ -42,5 +42,63 @@
     "FieldType": "GAWithOrWithoutNotice",
     "ElementLabel": " ",
     "SecurityClassification": "Public"
+  },
```

Comment:
Hey Smita, This isn't related to your change. I just found that from line 2 to 17 isn't been used anywhere. I guess we could remove it if that's make sense. Would you be fine removing those lines as part of your PR? Thanks

Reply:
We very much need this.... If you look at CaseFieldGAspec.json, we create collection of this item and populate it on about-to-submit callback....